src/styles/_summary.scss

```diff
@@ -33,6 +35,13 @@
   margin: 0.7rem auto;
 }
 
+/* Stack summary boxes vertically if container < ~1000px to avoid squishing */
```

Comment:
Much better! I don't think it's strictly required to explain what this is doing. I think it's quite obvious. The interesting point would be why 1000px exactly. I'll amend the comment and mention that there is no specific reason, it just fits.